src/nupic/algorithms/SpatialPooler.hpp

```diff
@@ -306,8 +308,8 @@ namespace nupic {
            */
           virtual void save(ostream& outStream) const;
 
-          virtual void write(ostream& stream) const;
-          virtual void write(SpatialPoolerProto::Builder& proto) const;
+          using Serializable::write;
```

Comment:
I tried that, and it didn't work. Here's why: http://stackoverflow.com/questions/2973976/inheritance-and-method-overloading